.kres.yaml

```diff
@@ -258,6 +258,34 @@ spec:
             artifactPath: _out
         - name: ci-temp-release-tag
         - name: lint
+    - name: grype-scan
+      buildxOptions:
+        enabled: true
+      depends:
+        - default
+      runners:
+        - self-hosted
+        - talos
+      steps:
+        - name: download-artifacts
+          artifactStep:
+            type: download
+            artifactName: talos-artifacts
+            artifactPath: _out
+        - name: ci-temp-release-tag
+        - name: local-grype-scan-result
+          environment:
+            DEST: _out
+        # Fail the run if found vulnerabilities which do not have a statement
```

Comment:
Here's an explanation:

Listed vulnerabilities have been acknowledged in https://github.com/siderolabs/talos-vex, so they do not fail the build. The report in artifacts, however, does list those, because they're `affected` or `under_investigation` and are of the interest to users. However, users are acknowledged of those by the advisories, so maintenance duties on the side of Talos are done (as those are tracked now) and the CI should pass again.

<img width="2415" height="646" alt="image" src="https://github.com/user-attachments/assets/f8096c6d-a94b-4a38-a864-bd3dd2b8571a" />